docs/reference/rollup/apis/rollup-api.asciidoc

```diff
@@ -61,15 +60,15 @@ Index to roll up. Cannot be a <<data-streams,data stream>> or
 <<indices-aliases,index alias>>. Does not support <<multi-index,multi-target
 syntax>> or wildcards (`*`).
 
-[role="child_attributes"]
-[[rollup-api-request-body]]
-==== {api-request-body-title}
-
 `rollup_index`::
 (Required, string)
 New index that stores the rollup results. Cannot be an existing index,
 a <<data-streams,data stream>>, or an <<indices-aliases,index alias>>.
-+
+
+[role="child_attributes"]
+[[rollup-api-request-body]]
+==== {api-request-body-title}
+
```

Comment:
figured I'd ping you @jrodewig because of this docs change :)

Reply:
Thanks for the ping @talevy. The `+` includes the other paragraph in the `<rollup_index>` def.

I pushed 4d0fa5ac5e18d5f5a199dade9f57755502ce6b21 to relocate the para and make some other minor fixes. However, this looks great overall!

Reply:
awesome. appreciate the fixes 💯 !